File: blimp/chunks/lang/benchmarks/spawn.py
import time
import threading

class Counter:
    def __init__(self):
        self.count = 0

    def increment(self):
        self.count += 1
        return self.count

start = time.time()
counters = []
for _ in range(100):
    c = Counter()
    c.increment()
    counters.append(c)
elapsed = time.time() - start
print(f"100 actors, 100 sends ({elapsed*1000:.1f}ms)")

File: blimp/chunks/lang/benchmarks/spawn.rs
use std::time::Instant;
use std::sync::mpsc;
use std::thread;

fn main() {
    let start = Instant::now();

    for _ in 0..100 {
        let (tx, rx) = mpsc::channel();
        let handle = thread::spawn(move || {
            // "Actor" receives increment message
            let mut count = 0i64;
            count += 1;
            tx.send(count).unwrap();
        });
        let _result = rx.recv().unwrap();
        handle.join().unwrap();
    }

    let elapsed = start.elapsed();
    println!("100 actors, 100 sends ({:.1}ms)", elapsed.as_secs_f64() * 1000.0);
}
